frontend/app/components/team-form.js

```diff
@@ -40,8 +41,11 @@ export default class AccountForm extends BaseFormComponent {
   handleSubmitSuccess(savedRecords) {
     this.abort();
     if (this.isNewRecord) {
-      let team = savedRecords[0].id;
-      this.router.transitionTo("teams.show", team);
+      let team = savedRecords[0];
+      let teams = this.navService.availableTeams.toArray();
```

Comment:
Not better if available teams in navService would be an Array?

Reply:
It is. But it's a MutableArray.
Problem with that is: Ember somehow doesn't like it if you manipulate the query result (Which navService.availableTeams is). So when I push into that, it throws an error. So I need to convert it to a JS Array, then push into that array and then overwrite the navService.availableTeams variable. It's stupid, I don't like it, but it's the only way I figured out how to get around it...

Reply:
And if you convert the Array when you initialize it? So that we have a JS Array instead of an Ember MutableArray?

Reply:
I could just keep it as a JS Array inside the navService... Would also clear confusion since it's sometimes a JS and sometimes a Mutable Array. Will do!

Reply:
Nice! Thanks 😄